contrib/scripts/codepage_to_utf8.py

```diff
@@ -0,0 +1,38 @@
+#!/bin/python3
+"""
+Convert file from legacy codepage to Unicode UTF-8
+"""
+import argparse
+import os
+import sys
+
+parser = argparse.ArgumentParser(description="Convert file from legacy codepage to Unicode UTF-8",
```

Comment:
Use pip to install 'black', which is a very robust (and opinionated! 😅) python formatter.

You can then execute 'black my script.py' and reformat it. 

This is what I've been using, and it works great.

Reply:
Done, although I'm not a fan of the way it split lines even further.